EIPS/eip-5202.md

```diff
@@ -48,7 +52,32 @@ A blueprint contract MAY insert any bytes (data or code) between the version byt
 
```

Comment:
note - EIP 3540 is referenced as a hypothetical, not as a dependency of this EIP, so this check probably needs to be overridden by a reviewer

Reply:
We don't allow references to less stable proposals because there's nothing stopping the author of such a proposal from completely changing its meaning. Today EIP-3540 could be the "Ethereum Object Format", but tomorrow it could be "Specifying Blueprints by Sending 1 ETH to samwilsn.eth".

You could do:

> - The Ethereum Object Format (EOF) could provide another...

Reply:
done in [72b3c68](https://github.com/ethereum/EIPs/pull/6847/commits/72b3c68c09feef50098bf1f7ac1669909079f964)